tests.py

```diff
@@ -11,7 +13,7 @@ def test_args(self):
         testargs = ["main", "-i", "my_test_input.txt",
```

Comment:
can we also mock `main.requests.get` for this unit? so that it won't really call the API every time the test runs

I have a plan to add Github Action later to run the test

Reply:
Do you mean doing a `setUp` for the case to apply the mock for all tests? I was unsure if that would be useful because we might want to mock different responses for other API related tests (and maybe create a different class for only API related tests too?).

Right now both `tests_args` and `tests_no_args` don't touch the API functions so they should be safe (unless there's something about GitHub Actions that I'm missing).

Reply:
oh my bad, I didn't see you're only testing the readArgs function, I thought it was testing the __main__

OK, it's good then, let's merge! :rocket: